src/test/java/gov/usgs/cida/pubs/dao/LinkTypeDaoIT.java

```diff
@@ -22,7 +22,7 @@
 	classes={DbTestConfig.class, LinkTypeDao.class})
 public class LinkTypeDaoIT extends BaseIT {
 
-	public static final int LINK_TYPES_CNT = 33;
+	public static final int LINK_TYPES_CNT = 38;
```

Comment:
I probably  missed this, I do not see where the link type count would have increase from
33 to 38.

Reply:
There was an earlier commit in pubswh-ui that increased the number